rpm/sofia-sip.spec

```diff
@@ -23,59 +26,56 @@ multimedia distribution, and multimedia conferences.
 
 
 %package glib
-Summary:    Glib bindings for Sofia-SIP
-Group:      Communications/Telephony and IM
+Summary:    Glib bindings for Sofia-SIP 
 Requires:   %{name} = %{version}-%{release}
-Requires(post): /sbin/ldconfig
-Requires(postun): /sbin/ldconfig
 
 %description glib
 GLib interface to Sofia SIP User Agent library.
 
 %package utils
 Summary:    Sofia-SIP Command Line Utilities
-Group:      Applications/Internet
 Requires:   %{name} = %{version}-%{release}
 
 %description utils
 Command line utilities for the Sofia SIP UA library.
 
 %package glib-devel
 Summary:    Glib bindings for Sofia SIP development files
-Group:      Development/Libraries
-Requires:   %{name} = %{version}-%{release}
-Requires:   sofia-sip-glib = %{version}-%{release}
-Requires:   sofia-sip-devel = %{version}-%{release}
+Requires:   %{name}-glib = %{version}-%{release}
+Requires:   %{name}-devel = %{version}-%{release}
 
 %description glib-devel
 Development package for Sofia SIP UA Glib library. This package
 includes libraries and include files for developing glib programs
 using Sofia SIP.
 
-
 %package devel
 Summary:    Sofia-SIP Development Package
-Group:      Development/Libraries
 Requires:   %{name} = %{version}-%{release}
+Requires:   pkgconfig
 
 %description devel
 Development package for Sofia SIP UA library.
 
+%package doc
+Summary:   Documentation for %{name}
+Requires:  %{name} = %{version}-%{release}
+
+%description doc
+Documentation for %{name}.
+
+
 %prep
-%setup -q -n %{name}-%{version}/src
+%autosetup -n %{name}-%{version}/%{name}
 
 %build
-%configure --disable-static
+%reconfigure --disable-static --without-doxygen --disable-stun
 make %{?_smp_mflags}
 
 %install
-rm -rf %{buildroot}
 %make_install
-
 find %{buildroot} -name \*.h.in -delete
-
-%check
```

Comment:
Should we not run the unit tests? In this case check would still be used.

Reply:
I think one of them failed but I can try again...

Reply:
OK, now I remember why I commented it out. One test fails like this in scratchbox:
```
[SB2 sdk-build SailfishOS-latest-aarch64] slava@Think sofia-sip $ ./libsofia-sip-ua/su/torture_su
torture_su.c:63: torture_su test_sockaddr() FAILED: su_getlocalinfo(hints, &res) != 0 or 18446744073709551615 != 0
[SB2 sdk-build SailfishOS-latest-aarch64] slava@Think sofia-sip $ gdb
```
and yet the same executable worked fine on the real hardware where I was planning to debug it. I assumed an emulator specific issue rather than a real problem and decided not to spend time on digging deeper.

Reply:
Even if these happened to pass in sb2 I'd not enable checks unless there are really good reasons to do so. We have the nightly tests running for where it matters.